### ISSUE TITLE
Core Pydantic V1 functionality isn't compatible with Python 3.14 or greater

### ISSUE THREAD

**Issue by bringert** [phillipdupuis/pydantic-to-typescript#64]
When running with Python 3.14, I get this warning:

```
pydantic2ts/pydantic_v1.py:2: UserWarning: Core Pydantic V1 functionality isn't compatible with Python 3.14 or greater.
  from pydantic.v1 import BaseModel, create_model  # type: ignore
```
